packages/contracts-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snickerdoodlelabs/contracts-sdk",
-  "version": "3.0.0",
+  "version": "3.0.2",
```

Comment:
I had to update contracts-sdk on develop. Please merge develop then update this accordingly